scripts/php/run-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+source ../setup-variables.sh
+
+bash ${PHP_SDK_GIT_REPO_FOLDER}/SalesForce/vendor/bin/phpunit -c ${PHP_SDK_GIT_REPO_FOLDER}/SalesForce/phpunit.xml.dist --testsuite=php-sdk-api,php-sdk-auth
+exit $?
+
+popd
```

Comment:
Is the `popd` command needed here?

Reply:
This script will no longer exist after all the PR's will be merged